app/src/ui/changes/no-changes.tsx

```diff
@@ -577,6 +618,19 @@ export class NoChanges extends React.Component<INoChangesProps, {}> {
     )
   }
 
+  public componentDidMount() {
+    this.transitionTimer = window.setTimeout(() => {
+      this.setState({ enableTransitions: true })
+      this.transitionTimer = null
+    }, 500)
+  }
+
+  public componentWillUnmount() {
+    if (this.transitionTimer !== null) {
+      clearTimeout(this.transitionTimer)
```

Comment:
Should `this.transitionTimer` also be set to `null` after the call to `clearTimeout`?

Reply:
It could but there's no reason to since the component is being unmounted. An unmounted component will not be reused so that value won't either.